src/app/components/PortraitVideoCarousel/PortraitVideoPromo/index.styles.ts

```diff
@@ -47,22 +47,25 @@ const styles = {
   gradientOverlay: ({ spacings }: Theme) =>
     css({
       position: 'absolute',
-      bottom: 0,
-      left: 0,
-      right: 0,
+      inset: 'auto 0 0 0',
       padding: `${pixelsToRem(28)}rem ${spacings.FULL}rem ${spacings.FULL}rem`,
       background:
         'linear-gradient(to bottom, rgba(0, 0, 0, 0) 0%, rgba(0, 0, 0, 0.65) 24%, rgba(0, 0, 0, 1) 100%)',
     }),
-  durationContainer: ({ palette }: Theme) =>
+  durationContainer: ({ palette, mq }: Theme) =>
     css({
-      display: 'flex',
+      display: 'inline-flex',
       alignItems: 'center',
       color: palette.WHITE,
+      [mq.FORCED_COLOURS]: {
```

Comment:
Great, would it make this content more readable in this mode, if this background was instead applied on the first nested div within the button so the duration and title don't look like seperate elements?